internal/cli/apps_test.go

```diff
@@ -11,15 +11,15 @@ import (
 	"gopkg.in/auth0.v5/management"
 )
 
-func TestClientsListCmd(t *testing.T) {
+func TestAppsListCmd(t *testing.T) {
 	// Step 1: Setup our client mock for this test. We only care about
 	// Clients so no need to bootstrap other bits.
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
 	clientAPI := auth0.NewMockClientAPI(ctrl)
 	clientAPI.EXPECT().
-		List().
+		List(gomock.Any()).
```

Comment:
creating a matcher for this could be really tricky because it is a function that takes a requests and `apply` the changes. I don't it will add a fundamental value to the test, but let me know